### PR TITLE
feat: new improved responsive layout

### PR DESCRIPTION
**Pull request by albertmir**
NOTE: I'm not a Tailwind expert, yet. So, please, feel free to suggest any improvements :)

### CODE REVIEW

File: src/pages/Home.vue [hunk 1]
```diff
@@ -47,7 +49,7 @@ export default {
     const currentYear = new Date().getFullYear()
 
     const { data, error, mutate, isValidating } = useSWRFetch(
-      `https://api.rawg.io/api/games?dates=${currentYear}-01-01,${currentYear}-12-31&ordering=-rating`,
+      `https://api.rawg.io/api/games?dates=${currentYear}-01-01,${currentYear}-12-31&ordering=-rating&page_size=24`,
```
Comment (albertmir) [inline src/pages/Home.vue:52]: Setting the `page_size` to 24 in order to fill all the lines with cards 😉

File: src/pages/Home.vue [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
-          class="flex-1 text-black px-4 py-2 bg-gray-800 text-gray-300 rounded-sm"
-          placeholder="Enter your favorite game..."
-        />
-        <button
-          @click="fetchGame"
-          class="py-2 px-5 uppercase font-bold bg-green-400 rounded-sm ml-2"
-        >
-          Search
-        </button>
+  <section class="container mx-auto grid sm:px-8 sm:py-12 sm:gap-x-8 md:py-16">
```
Comment (jeferson-sb) [inline src/pages/Home.vue:2]: @albertmir the container looks a bit large to me at >1440px 
Could set a max-width to it? something like: `xl:max-w-4xl`

File: src/pages/Home.vue [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
+    <div class="">
```
Comment (jeferson-sb) [inline src/pages/Home.vue:3]: @albertmir do we really need this extra div em empty class?

File: src/pages/GameProfile.vue [hunk 4]
```diff
@@ -20,82 +25,144 @@
           />
         </svg>
       </router-link>
-      <h1 class="text-5xl font-bold">{{ gameDetails.name }}</h1>
-      <p class="opacity-75 mb-5">Release Date: {{ dateReleased }}</p>
-      <p
-        class="text-4xl font-semibold border-solid border-4 border-blue-600 w-24 text-center"
+      <h2
+        class="text-xl font-semibold text-white sm:text-2xl sm:leading-7 md:text-4xl"
```
Comment (jeferson-sb) [inline src/pages/GameProfile.vue:29]: @albertmir Can we increase the font-size a little bit on md?

File: src/pages/GameProfile.vue [hunk 5]
```diff
@@ -20,82 +25,144 @@
...
+          <div
+            class="flex items-center mr-2"
+            v-for="developer in gameDetails.developers"
+            :key="developer.id"
+          >
+            <div class="ml-1">
+              <img
+                v-if="developer.image_background"
+                :src="developer.image_background"
+                alt=""
```
Comment (jeferson-sb) [inline src/pages/GameProfile.vue:44]: Missing alt text

Comment (albertmir) [reply src/pages/GameProfile.vue:44]: Nice catch @jeferson-sb :)

File: src/pages/Home.vue (hunk 3 above)
Comment (albertmir) [reply src/pages/Home.vue:3]: Ooops. Not necessary 😓 Removing it...

File: src/pages/GameProfile.vue (hunk 4 above)
Comment (albertmir) [reply src/pages/GameProfile.vue:29]: Sure. Setting as **5xl**.

File: src/pages/Home.vue (hunk 2 above)
Comment (albertmir) [reply src/pages/Home.vue:2]: Tried with `xl:max-w-4xl` but I prefer the `md:max-w-5xl`. _4xl_ was a little bit small.

Comment (jeferson-sb) [reply src/pages/Home.vue]: Looks cool!

**Review by jeferson-sb — APPROVED** [review]

**Comment by jeferson-sb** [comment]
@albertmir Thanks for your amazing contribution 🎉🎊

**Comment by albertmir** [comment]
You're welcome @jeferson-sb! Thanks to you for offering this useful base to show the new concepts of Vue 3 😃

**Pull request merged**